### PR TITLE
[type:Integration Test] Add metrics-plugin Integration Test

### PR DESCRIPTION
**Pull request by erdengk**
Add metrics-plugin Integration Test

### CODE REVIEW

File: shenyu-integrated-test/shenyu-integrated-test-http/src/main/resources/application.yml [hunk 1]
```diff
@@ -33,6 +33,14 @@ management:
       enabled: false
 
 shenyu:
+  metrics:
+    enabled: true
+    name: prometheus
+    host: 127.0.0.1
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-http/src/main/resources/application.yml:39]:
```suggestion
    host: 0.0.0.0
```

File: shenyu-integrated-test/shenyu-integrated-test-http/config/prometheus.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+global:
+  scrape_interval: 5s
+
+scrape_configs:
+  - job_name: 'Apache ShenYu'
+    static_configs:
+      - targets: ['127.0.0.1:8090']
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-http/config/prometheus.yml:23]:
```suggestion
      - targets: ['shenyu-integrated-test-http:8090']
```

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: shenyu-integrated-test/shenyu-integrated-test-http/src/main/resources/application.yml (hunk 1 above)
Comment (erdengk) [reply shenyu-integrated-test/shenyu-integrated-test-http/src/main/resources/application.yml:39]: ok

File: shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java [hunk 3]
```diff
@@ -264,4 +265,25 @@ public Response getHttpService(final String url, final Map<String, Object> heade
...
+     * @param headers headers
+     * @param path    path
+     * @param type    type of response passed to {@link Gson#fromJson(String, Type)}
+     * @return response from gateway
+     * @throws IOException IO exception
+     */
+    public <S> S getHttpService(final String path, final Map<String, Object> headers, final Type type) throws IOException {
+        Response response = getHttpService(path, headers);
+        String respBody = Objects.requireNonNull(response.body()).string();
+        LOG.info("testMetricsPluginFromGateway({}) resp({})", path, respBody);
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java:282]: Pls modify the log, this method should be a common util.

File: shenyu-integrated-test/shenyu-integrated-test-http/config/prometheus.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+global:
+  scrape_interval: 5s
+
+scrape_configs:
+  - job_name: 'Apache ShenYu'
+    static_configs:
+      - targets: ['shenyu-integrated-test-http:8090']
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:9090"]
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-http/config/prometheus.yml:26]: Add a blank line at the end of the file

File: shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml [hunk 5]
```diff
@@ -87,13 +87,30 @@ services:
...
+      shenyu-integrated-test-http:
+        condition: service_healthy
+    networks:
+      - shenyu
+    volumes:
+      - type: bind
+        source: ./config/prometheus.yml
+        target: /etc/prometheus/prometheus.yml
+
+
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml:113]: remove meaningless blank lines

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java (hunk 3 above)
Comment (erdengk) [reply shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java:282]: ok

**Review by JooKS-me — APPROVED** [review]

**Pull request merged**